react-state-3/react-state-3_adding-animals/src/App.jsx

```diff
@@ -25,7 +25,7 @@ export default function App() {
   const [animals, setAnimals] = useState(initialAnimals);
 
   function handleAddAnimal(newAnimal) {
-    console.log(newAnimal);
+    setAnimals([newAnimal, ...animals]);
```

Comment:
You could also install uid package and add it in the code.
i.e. setAnimals([...animals, { id: uid(), ...newAnimal }])